includes/ets_user_qa_function.php

```diff
@@ -188,7 +189,7 @@ public function ets_ask_qustion_tab() {
 				<input type="hidden" id="productlength" class="productlength" name="Product_Qa_Length" value="<?php echo $productQaLength ?>">  
 				<input type="hidden" id="producttitle" name="ets_Product_Title" value="<?php echo $productTitle ?>"> 
 			</form>
-			<div id="ets_product_qa_length"><p></p></div> 
+			
```

Comment:
What was the use of statement `<div id="ets_product_qa_length"><p></p></div>` removed from this line.

Reply:
It is used when we put the shortcode and default functionality together when we don't use this it dismisses the shortcode load more.